src/components/VHeader/VHeader.vue

```diff
@@ -1,22 +1,64 @@
+        />
```

Comment:
Perhaps we can extract this path to an asset like the `VCheckbox`'s [checkmark](https://github.com/WordPress/openverse-frontend/blob/main/src/components/VCheckbox.vue#L23)? That looked neat!

Reply:
Done! Great idea.